tests/flow/tests_commands.py

```diff
@@ -165,14 +165,14 @@ def test_keys_syntax(env):
     check_error_message(env, con, "KEYS scope must be provided first for AI.SCRIPTEXECUTE command",
                         "AI.SCRIPTEXECUTE script{1} bar INPUTS 2 a{1} a{1}")
 
-    # ERR KEYS section in an inner AI.SCRIPTEXEUTE command within a DAG is not allowed.
-    check_error_message(env, con, "Already encountered KEYS scope in current command",
-                        "AI.DAGEXECUTE KEYS 1 a{1} |> AI.SCRIPTEXECUTE script{1} bar KEYS 1 a{1}")
+    # # ERR KEYS section in an inner AI.SCRIPTEXEUTE command within a DAG is not allowed.
+    # check_error_message(env, con, "Already encountered KEYS scope in current command",
+    #                     "AI.DAGEXECUTE KEYS 1 a{1} |> AI.SCRIPTEXECUTE script{1} bar KEYS 1 a{1}")
 
 
 # Todo: this test should change once the script store command is implemented.
```

Comment:
remove comment...
Should add here the scriptstore errors test (like modelstore)